# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/moz.build
# Commit: 1c0d9fdb8a06
# Full Hash: 1c0d9fdb8a06724c0166a9e08578e81952161b67
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2025-06-19 21:40:31
# Regressor Bug: 1968812
# File Overlap Count: 1
# Description:
#   Bug 1968812 - Remove the CallWorker thread and use TaskQueue::Observer instead. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D252686
# ==============================================================================

diff -r a3b51595f6d1 -r 1c0d9fdb8a06 dom/media/webrtc/libwebrtcglue/moz.build
--- a/dom/media/webrtc/libwebrtcglue/moz.build	Thu Jun 19 08:18:29 2025 +0000
+++ b/dom/media/webrtc/libwebrtcglue/moz.build	Thu Jun 19 08:18:29 2025 +0000
@@ -20,7 +20,6 @@
 
 UNIFIED_SOURCES += [
     "AudioConduit.cpp",
-    "CallWorkerThread.cpp",
     "FrameTransformer.cpp",
     "FrameTransformerProxy.cpp",
     "GmpVideoCodec.cpp",
@@ -34,6 +33,7 @@
     "WebrtcGmpVideoCodec.cpp",
     "WebrtcMediaDataDecoderCodec.cpp",
     "WebrtcMediaDataEncoderCodec.cpp",
+    "WebrtcTaskQueueWrapper.cpp",
     "WebrtcVideoCodecFactory.cpp",
 ]
 